### PR TITLE
Adding more operators to the docs

### PR DESCRIPTION
**Pull request by mneedham**
* Expand Into
* Argument
* Apply

### CODE REVIEW

File: community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala [hunk 1]
```diff
@@ -220,6 +220,26 @@ class QueryPlanTest extends DocumentingTestBase {
     )
   }
 
+  @Test def argument() {
+    profileQuery(
+      title = "Argument",
+      text =
+        """The Argument operator is used in conjunction with Apply.
+          |It receives rows from Apply, processes them and then sends the result back.
```
Comment (systay) [inline community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala]: `It receives rows from Apply and yields them to the operator feeding off of Argument.`

The processing is done by the operators between Argument and Apply. In this plan, Apply feeds the resulting rows from Expand-Selection to the bottom of the RHS. Then it pulls data from the Optional operator, which ultimately pulls data from Argument, through the intermediate operators.

Argument/Apply are special in that you need to understand both. Can't have one without the other. Maybe we need a special little section for this?

**Comment by systay** [comment]
@mneedham bump

**Comment by boggle** [comment]
Could you please use the "NOT READY FOR MERGE" label on these docs PRs? I'm never sure when to pick them up.

**Comment by systay** [comment]
How about we re-open this once it's ready for merging?

**Pull request closed without merging**